### DISCUSSION TITLE
BACKUP to s3 repo failed

### DISCUSSION THREAD

**Discussion by lymaimei** [apache/doris#22110] [A - General / Q&A] [answered]
backup in following steps
https://doris.apache.org/docs/dev/admin-manual/data-admin/backup/

1.create repo
![image](https://github.com/apache/doris/assets/19678288/641e9ac2-2a99-4217-b112-71d0b2d3b5c4)

repo was created,repo file can be seen in aws s3
![image](https://github.com/apache/doris/assets/19678288/5b26b35c-4995-4944-b2c4-7e7a0cdd6128)

2.back up use follow sql
BACKUP SNAPSHOT snapshot_test TO s3_repo ON (TS_S3_TEST) PROPERTIES ("type" = "full");
get error msg
[INVALID_ARGUMENT]S3 properties are incorrect, please check properties.]

many thanks

**Comment by hf200012** [comment, marked as answer]
```
CREATE REPOSITORY `s3_repo`
WITH S3
ON LOCATION "s3://bucket_name/test"
PROPERTIES
(
    "AWS_ENDPOINT" = "http://xxxx.xxxx.com",
    "AWS_ACCESS_KEY" = "xxxx",
    "AWS_SECRET_KEY"="xxx",
    "AWS_REGION" = "xxx"
); ```
